packages/tooltip/src/lib/Tooltip.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { forwardRef, ReactElement } from 'react';
+import { Tooltip as MuiTooltip, TooltipProps as MuiTooltipProps } from '@mui/material';
+
+export type TooltipProps = MuiTooltipProps;
+
+export const Tooltip = forwardRef<ReactElement, TooltipProps>((props: TooltipProps, forwardedRef) => {
+  let { children, arrow, ...rest } = props;
+
+  if (arrow !== false) {
```

Comment:
This seems to be a little backwards.
`<Thing />` - Shows the arrow
`<Thing arrow />` - Shows the arrow
`<Thing arrow={false} />` - Doesn't show the arrow

Is the intention to only ever show the arrow for our tooltips?

Reply:
Tyler requested we make this property default to true, out of the box it defaults to false.

Reply:
This has been updated to set arrow to true internally and disallow the user from modifying it.

Reply:
Good catch, omitted